tools/rename_apk_aab.sh

```diff
@@ -0,0 +1,111 @@
+#!/bin/bash
+
+# This script aims to rename any `.apk` or `.aab` file to `[wpandroid|jpandroid]-{version}[-Signed].apk/aab`
+# depending on the APK/AAB package name, version name and package signature.
+#
+# The main use of this tool os to easily rename the signed `.apk` and/or original `.aab` downloaded from the PlayStore
+# (and re-signed by Google depending on the context), as when you download those their basename is usually just the
+# versionCode and whose name doesn't distinguish from a binary corresponding to WordPress vs Jetpack.
+#
+# By running this script on the APK or AAB files provided – defaulting to all the APK and AAB files in `~/Downloads` if
+# no parameter provided – the files will be renamed with a basename appropriate to the ones we use when attaching
+# those files to the GitHub release.
+
+### Input Parameters ###
+
+# Usage:
+#   rename_apk_aab.sh [file1.apk] [file2.aab] [...]
+#
+# Without any parameter, applies the rename to every APK/AAB file found in ~/Downloads
+#
+INPUT_FILES=( "$@" )
+if [[ $# -lt 1 ]]; then
+  IFS=$'\n'
+  INPUT_FILES=( $(find ~/Downloads -name *.apk -o -name *.aab) )
+  unset IFS
+fi
+
+
+##################################
+### Path to Android tools used ###
+##################################
+AAPT2=$(ls -1t $ANDROID_SDK_ROOT/build-tools/*/aapt2 | head -n1)
+APKSIGNER=$(ls -1t $ANDROID_SDK_ROOT/build-tools/*/apksigner | head -n1)
```

Comment:
We typically have multiple versions of the build-tools installed (e.g. `30.0.0`, `30.0.2`, `29.0.2`…) especially if we have installed multiple SDK. This picks the ones with the most-recent `mtime` (`-t`)